### DISCUSSION TITLE
How to call dask from c++

### DISCUSSION THREAD

**Discussion by Manoj-red-hat** [dask/dask#9742] [Q&A]
Hi Team,

I am trying to offload some computation from my database to Dask using UDF. UDF is written in C++, now I am wondering how to call Dask client and submit job to running Dask cluster ?

Thanks

**Comment by ncclementi** [comment]
@Manoj-red-hat as the discussions section mentions, you would have a better chance of getting an answer by posting on the dask discourse
https://dask.discourse.group/
